README.md

```diff
@@ -281,14 +281,13 @@ index e93321f..7ac4fe2 100644
  
    batch_size = FLAGS.batch_size
    num_epochs = FLAGS.num_epochs
-+  num_models = jax.device_count()
  
 -  model = create_model(rng)
 -  optimizer = create_optimizer(model, FLAGS.learning_rate, FLAGS.momentum)
-+  optimizers = create_optimizers(random.split(rng, num_models))
++  optimizers = create_optimizers(random.split(rng))
```

Comment:
how many splits are supposed to be created here? I would guess num_devices?

Reply:
Yes good point, I updated it.